ZelBack/src/services/appsService.js

```diff
@@ -389,6 +389,20 @@ async function nodeTier() {
   throw new Error('Unrecognised Flux Node tier');
 }
 
+async function isNodeStatusConfirmed() {
+  const response = await daemonService.getZelNodeStatus();
```

Comment:
do we want to throw an error or just simple true false function?

Reply:
saw that you already resolved all the situations :D